TKMovesets2/GameSpecific/Structures/Structs_t8.h

```diff
@@ -311,72 +369,68 @@ namespace StructsT8
 
 	// -- Other -- //
 
-	struct MovesetTable //todo
+	struct MovesetTable
 	{
-		union {
-			struct {
-				Reactions* reactions;
-				uint64_t reactionsCount;
+		Reactions* reactions;
+		uint64_t _0x8; // what if this was wrongly placed?
+		uint64_t reactionsCount;
 
-				Requirement* requirement;
-				uint64_t requirementCount;
+		Requirement* requirement;
+		uint64_t requirementCount;
 
-				HitCondition* hitCondition;
-				uint64_t hitConditionCount;
+		HitCondition* hitCondition;
+		uint64_t hitConditionCount;
 
-				Projectile* projectile;
-				uint64_t projectileCount;
+		Projectile* projectile;
+		uint64_t projectileCount;
 
-				Pushback* pushback;
-				uint64_t pushbackCount;
+		Pushback* pushback;
+		uint64_t pushbackCount;
 
-				PushbackExtradata* pushbackExtradata;
-				uint64_t pushbackExtradataCount;
+		PushbackExtradata* pushbackExtradata;
+		uint64_t pushbackExtradataCount;
 
-				Cancel* cancel;
-				uint64_t cancelCount;
+		Cancel* cancel;
+		uint64_t cancelCount;
 
-				Cancel* groupCancel;
-				uint64_t groupCancelCount;
+		Cancel* groupCancel;
+		uint64_t groupCancelCount;
 
-				CancelExtradata* cancelExtradata;
-				uint64_t cancelExtradataCount;
+		CancelExtradata* cancelExtradata;
+		uint64_t cancelExtradataCount;
 
-				ExtraMoveProperty* extraMoveProperty;
-				uint64_t extraMovePropertyCount;
+		ExtraMoveProperty* extraMoveProperty;
+		uint64_t extraMovePropertyCount;
 
-				OtherMoveProperty* moveBeginningProp; // Extraprops that play when a move starts
-				uint64_t moveBeginningPropCount;
+		OtherMoveProperty* moveBeginningProp; // Extraprops that play when a move starts
+		uint64_t moveBeginningPropCount;
 
-				OtherMoveProperty* moveEndingProp; // Extraprops that play when a move ends
-				uint64_t moveEndingPropCount;
+		OtherMoveProperty* moveEndingProp; // Extraprops that play when a move ends
+		uint64_t moveEndingPropCount;
 
-				Move* move;
-				uint64_t moveCount;
+		Move* move;
+		uint64_t moveCount;
 
-				Voiceclip* voiceclip;
-				uint64_t voiceclipCount;
+		Voiceclip* voiceclip;
+		uint64_t voiceclipCount;
 
-				InputSequence* inputSequence;
-				uint64_t inputSequenceCount;
+		InputSequence* inputSequence;
+		uint64_t inputSequenceCount;
 
-				Input* input;
-				uint64_t inputCount;
+		Input* input;
+		uint64_t inputCount;
 
-				UnknownParryRelated* unknownParryRelated;
-				uint64_t unknownParryRelatedCount;
+		UnknownParryRelated* unknownParryRelated;
+		uint64_t unknownParryRelatedCount;
 
-				CameraData* cameraData;
-				uint64_t cameraDataCount;
+		CameraData* cameraData;
+		uint64_t cameraDataCount;
 
-				ThrowCamera* throwCameras;
-				uint64_t throwCamerasCount;
-			};
-			struct {
-				void* listAddr;
-				uint64_t listCount;
-			} entries[19];
-		};
+		ThrowCamera* throwCameras;
+		uint64_t throwCamerasCount;
+
+		UnknownNew* _unkown_0x298;
+		uint64_t _unkown_0x298Count;
```

Comment:
Try to keep the whole union stuff here, just remove the Reactions from it because of the wrong placed 8 bytes, this list is iterated on many times in the code and it saves a lot of lines